src/ducks/SANCharts/ChartMetrics.js

```diff
@@ -32,6 +38,15 @@ class ChartMetrics extends Component {
     )
   }
 
+  componentWillReceiveProps (nextProps, nextContext) {
```

Comment:
`componentWillReceiveProps` is deprecated. Please, read the [documentation](https://ru.reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops)